.github/ISSUE_TEMPLATE/update_gtfs.md

```diff
@@ -0,0 +1,25 @@
+---
+name: GTFS Data
+about: Update GTFS data in agencies.yml
+title: 'GTFS Data: '
+labels: ''
+assignees: '@evansiroky'
+---
+
+Update `agencies.yml` in order to....
+
+Add the following gtfs datasets:  
+
+- ...
+- ...
+
+Update the following gtfs datsets:  
+
+- ...
+- ...
+
+## Checklist
+
+- [ ] Made sure any new feeds have `itp_id`s that are not duplicative
+- [ ] Confirmed URIs are valid  
```

Comment:
Can you add a note that this should be done manually and that ideally it will also be part of an automated check as described in https://github.com/cal-itp/data-infra/issues/908? Same goes for the PR template.